### PR TITLE
Fixed Gramatical errors and spelling mistakes in CONTRIBUTING.md and README.md

### PR DESCRIPTION
**Pull request by sree98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 
 The goal of Ready Responder is to offer volunteer groups a program that allows them to track their resources and personnel, especially during emergencies or multi-day events.  This application might be used by volunteer firefighters, auxiliary police, Medical Reserve Corp (MRC), CERT organizations, amateur radio operators (ARES/RACES), church based relief groups, shelter managers or even science-fiction conventions.
 
 #### Current Features:
-* Web-based user interface, available from both desktop and mobile
-* Tracks complete data of personnel, including attendance, responsiveness and training
-* Tracks equipment, including serial numbers, sources, grants and service records
+* Web-based user interface, available from both desktop, and mobile
```
Comment (kgf) [inline README.md]: Is the comma correct in a list that only has two items ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,23 +3,23 @@ Contributing to ReadyResponder
...
 
 Many people contribute to ReadyResponder by writing some code! Here's how:
 
-1. [Fork](https://help.github.com/articles/fork-a-repo) ReadyResponder
-2. Create a topic branch - `git checkout -b my_branch`
-3. Make your changes
-4. Commit your changes to your branch
-3. Push your branch to your github repo - `git push origin my_branch`
+1. [Fork](https://help.github.com/articles/fork-a-repo) ReadyResponder.
+2. Create a topic branch. - `git checkout -b my_branch`
```
Comment (kgf) [inline CONTRIBUTING.md]: Would you be willing to add a line about cloning your fork to your development machine ?

**Comment by kgf** [comment]
Thanks for your help ! Good documentation is ... good !

File: README.md (hunk 1 above)
Comment (sree98) [reply README.md]: I am sorry that was my mistake. It isn't correct.

File: CONTRIBUTING.md (hunk 2 above)
Comment (sree98) [reply CONTRIBUTING.md]: Would this be a good line for that purpose?

Clone to your development machine (git clone git@github.com:<account>/<projectname>.git)

Comment (kgf) [reply CONTRIBUTING.md]: Thanks, yes that could be good.
Could we give them a link to github docs, like line 16 ?

File: README.md (hunk 1 above)
Comment (kgf) [reply README.md]: Oh good, I was able to salvage a shred of my pride after having all the other errors. 😁

File: CONTRIBUTING.md (hunk 2 above)
Comment (sree98) [reply CONTRIBUTING.md]: What would the link be, in that case?

File: README.md (hunk 1 above)
Comment (sree98) [reply README.md]: HAHA. Don't worry these errors are pretty common, and after all, we aren't English majors, are we?

File: CONTRIBUTING.md (hunk 2 above)
Comment (kgf) [reply CONTRIBUTING.md]: How about this
https://help.github.com/articles/cloning-a-repository/

Comment (sree98) [reply CONTRIBUTING.md]: How about this:

[Clone](https://help.github.com/articles/cloning-a-repository/) the forked repository to your development or local machine.

The line for that is `[Clone](https://help.github.com/articles/cloning-a-repository/) the forked repository to your development or local machine.`

Comment (kgf) [reply CONTRIBUTING.md]: That would be a great addition. Once thta's there, I'm ready to merge.
Thanks !

**Comment by kgf** [comment]
Thanks for your help ! We appreciate it.

**Pull request merged**